### DISCUSSION TITLE
Engram on Split 40%

### DISCUSSION THREAD

**Discussion by Tristen-Sinanju** [binarybottle/engram-layout#44] [Show and tell]
over the past few days i worked on a Lunakey Mini project, as well as the writing a keymap and sorting out the OLED Drivers
anyhow, i thought i'd share since the 3x6_4 layout is popular

keymap here, includes dependencies - [tristen.zip](https://github.com/binarybottle/engram/files/8808824/tristen.zip)

![282262140_998766570834221_1108761804748278285_n](https://user-images.githubusercontent.com/84834000/171271077-031682d5-c848-4e69-a308-fc69e766835c.jpg)

https://user-images.githubusercontent.com/84834000/171271104-0f7eb686-e1bc-4bdd-b8c8-70f11d14b475.mp4

**Comment by binarybottle** [comment]
Cool -- Thank you for sharing, @Tristen-Sinanju!
